test/cppsuite/test_harness/workload_validation.h

```diff
@@ -36,6 +36,23 @@ extern "C" {
 }
 
 namespace test_harness {
+
+/* A key has an associated value if it exists. */
+struct Key {
+    bool exists;
+    std::string value;
+};
+
+/* A collection is made of Key objects. */
+struct Collection {
+    std::map<std::string, Key> keys;
```

Comment:
Should this be `std::map<key_value_t, Key> keys`?

Reply:
Correct